documentation/api/index.html

```diff
@@ -88,22 +88,45 @@
         instance (both locally and on AWS), Flyway will automatically use the JDBC url, user and password
         <a href="https://boxfuse.com/docs/databases#envvars">provided by Boxfuse</a>.</div>
 
-    <h2>Programmatic Configuration (Android)</h2>
+    <h2 id="android">Android Programmatic Configuration (Android Studio)</h2>
 
-                <p>Start by adding the <a href="https://github.com/SQLDroid/SQLDroid">SQLDroid</a> driver jar to the
-                    <code>libs</code> directory</p>
+    <p>In order to use Flyway on Android you have to add flyway-core as well as SQLDroid as dependencies. There are two things to keep in mind with Android: First, you have to load the migrations as <i>assets</i>, not <i>resources</i> and second, you have to let Flyway know your Android context, by calling <code>ContextHolder.setContext</code>.</p>
+
+    <p>1. Add the necessary dependencies to <code>build.gradle</code>:</p>
+
+    <pre class="prettyprint">dependencies {
+    // Your other dependencies
+    // ...
+
+    compile 'org.flywaydb:flyway-core:{{ site.flywayVersion }}'
+    compile 'org.sqldroid:sqldroid:1.0.3'
+}</pre>
+
+    <p>2. Make source that your migrations are included as assets (notice that assets have to be declared in the project itself and not in a dependency. But you can use reference e.g. <code>'../lib/src/main/resources'</code> to use the resources of a lib project)</p>
+
+<pre class="prettyprint">android {
+    // SDK, config, buildTypes, etc
+    // ...
+
+    sourceSets {
+        // Place your db/migration folder here
+        main { assets.srcDirs = ['src/main/assets'] }
+    }
+
+    // ...
+}</pre>
+
+    <p>3. Include the setup in your main activity onCreate or application onCreate:</p>
 
-                <p>Then inside your main activity add this:</p>
                 <pre class="prettyprint">import org.flywaydb.core.Flyway;
 import org.flywaydb.core.api.android.ContextHolder;
-import android.database.sqlite.SQLiteDatabase;
+import org.sqldroid.DroidDataSource;
 
```

Comment:
Both will work. I suggest this variant because it makes explicit the connection between the database and Flyway. It helped me explain to colleagues what is going on. Which variant to prefer depends on how you want to use SqlDroid further.